File: ympa_fsharp_functions_server/src/model/BusinessSettingsDTO.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.CurrencyType

module BusinessSettingsDTO =

  //#region BusinessSettingsDTO

  [<CLIMutable>]
  type BusinessSettingsDTO = {
    [<JsonProperty(PropertyName = "onlyDefaultPrice")>]
    OnlyDefaultPrice : bool;
    [<JsonProperty(PropertyName = "currency")>]
    Currency : CurrencyType;
  }

  //#endregion

File: src/main/scala/org/openapitools/server/model/BusinessSettingsDTO.scala
package org.openapitools.server.model


/**
 * Настройки кабинета.
 *
 * @param onlyDefaultPrice Можно ли установить только [базовую цену](*rule): * `false` — можно задать и базовую цену, и цены в конкретных магазинах. * `true` — можно задать только базовую цену.  for example: ''null''
 * @param currency  for example: ''null''
*/
final case class BusinessSettingsDTO (
  onlyDefaultPrice: Option[Boolean] = None,
  currency: Option[CurrencyType] = None
)

